edit/logging.banzaicloud.io.output/providers/utils.js

```diff
@@ -0,0 +1,24 @@
+export const protocol = ['http', 'https'];
+
+export function updatePort(setter, port) {
+  // We set the value to 0 then the actual value because if we exceed the maximum of
+  // 65535 all subsequent values will continue to return 65535 which vue ignores and
+  // allows the user to continue appending values in the input even the the stored value
+  // remains 65535.
+  setter(0);
+  setter(normalizePort(port));
+}
+
+export function normalizePort(port) {
+  const portAsInt = Number.parseInt(port);
```

Comment:
Apparently this isn't _actually_ a problem in modern ecmascript versions, but seeing a parseInt without a radix triggers PTSD of strings with leading zeros getting parsed as octal; `parseInt(string, 10)`